### PR TITLE
Screenshot

### PR DESCRIPTION
**Pull request by jburel**
* Add screenshot to the notebook
* Check that you can run the notebook. The first cell loading the library should be enough since it highlights the problem due itkwidgets

cc @pwalczysko

### CODE REVIEW

File: notebooks/idr0062_pixel_classification_zarr.ipynb [hunk 1]
```diff
@@ -727,6 +727,13 @@
     "compare(first_channel, data[0, 0, :, :, :], shadow=False, gradient_opacity=0.2, ui_collapsed=True)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![Compare](images/ilastik_1.png)"
```
Comment (pwalczysko) [inline notebooks/idr0062_pixel_classification_zarr.ipynb:734]: I cannot see that screenshot in the rendered notebook. Is that maybe not added ? Do not see that png added in the diff of the PR

**Comment by pwalczysko** [comment]
Works fine except https://github.com/ome/omero-guide-ilastik/pull/30/files#r631859311

File: notebooks/idr0062_pixel_classification_zarr.ipynb (hunk 1 above)
Comment (jburel) [reply notebooks/idr0062_pixel_classification_zarr.ipynb:734]: Sorry I forgot to push the images
Done

**Comment by pwalczysko** [comment]
Thank you, the screenshots are there now.
If I may suggest not taking the top part of the dock in Mac for ilastik_1.png though

Screenshot 2021-05-13 at 16 06 54

Also, in the meantime, I have a repeated error when built locally.

```
Pip subprocess error:
ERROR: Could not install packages due to an EnvironmentError: [Errno 28] No space left on device: '/tmp/pip-install-ef5bu1zn/itk-filtering'



CondaEnvException: Pip failed
```

I have 112 GB free space on my Mac, so not sure why is this happening.


Edit:
Trying now after cleanup with
``docker image prune -a ``

**Comment by jburel** [comment]
you will need to clear space
``docker system prune``
and 
``docker image prune -a``

**Comment by pwalczysko** [comment]
> you will need to clear space
> `docker system prune`
> and
> `docker image prune -a`

@jburel Thank you, that worked, also locally the notebook is running without problems now.

The screenshot improvement looks good as well.

Ready to merge fmpov

**Pull request merged**